src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -405,14 +467,13 @@ def create(dataset, target, feature, max_iterations=10,
 
         # Calculate training metric
         for data, label in train_data:
-            outputs = custom_NN.predict(data)
-            train_metric.update(label, outputs)
+            result = custom_NN.evaluate(data, label)
```

Comment:
Why did we change from `predict` to `evaluate`? Doesn't that break compatibility with the MXNet implementation (which we haven't deleted yet?)

Reply:
`predict` returns prediction probabilities. `evaluate` returns prediction probabilities and accuracies.

This will break with MXNet since we don't have `evaluate()` in `MultiLayerPerceptronMXNetModel()`